boards/x220/x220.config

```diff
@@ -35,3 +61,7 @@ export CONFIG_BOOT_KERNEL_REMOVE="quiet"
 export CONFIG_BOOT_DEV="/dev/sda1"
 export CONFIG_BOARD_NAME="ThinkPad X220"
 export CONFIG_FLASHROM_OPTIONS="--force --noverify-all -p internal:ich_spi_mode=hwseq --ifd --image bios"
```

Comment:
Same here

We do not just write BIOS region with those boards, and that should come with a big fat warning with osresearch.net changes. We want to write the whole ROM internally as we do externally.

`--ifd --image bios` not desired